backend/plugin/advisor/pg/advisor_statement_no_select_all.go

```diff
@@ -96,46 +96,17 @@ func (r *noSelectAllRule) handleSimpleSelectPramary(ctx *parser.Simple_select_pr
 		for _, target := range allTargets {
 			// Check if target is a Target_star (SELECT *)
 			if _, ok := target.(*parser.Target_starContext); ok {
-				// Find the top-level statement context to get the full statement text
-				var stmtCtx antlr.ParserRuleContext
-				parent := ctx.GetParent()
-				for parent != nil {
-					// Look for top-level statement contexts
-					switch p := parent.(type) {
-					case *parser.SelectstmtContext:
-						if isTopLevel(p.GetParent()) {
-							stmtCtx = p
-						}
-					case *parser.InsertstmtContext:
-						if isTopLevel(p.GetParent()) {
-							stmtCtx = p
-						}
-					}
-					if stmtCtx != nil {
-						break
-					}
-					parent = parent.GetParent()
+				stmtText := getTextFromTokens(r.tokens, ctx)
```

Comment:
Is this change expected?

Reply:
yes, get the full statement text is meaningless. For the statement below:
```sql
SELECT a FROM (SELECT * FROM t) subquery_table;
```
Previously the error message is `SELECT a FROM (SELECT * FROM t) subquery_table;
 uses SELECT ALL`, now is `(SELECT * FROM t) uses SELECT ALL`. Less noise.